src/datacollector/DataCollectionAttachmentManager.cs

```diff
@@ -196,86 +151,47 @@ public void AddAttachment(FileTransferInformationExtension fileTransferInfo)
             }
         }
 
-        /// <summary>
-        /// Dispose event object
-        /// </summary>
+        /// <inheritdoc/>
         public void Dispose()
```

Comment:
Why is this required? Do we not allocate a new `DataCollectionAttachmentManager` for data collection session?